<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>629</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>629</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>629</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_error">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>0</y>
     <width>1271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 66, 53);
border-radius: 5px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_close">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>10</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	
	background-image: url(:/close/cil-x.png);
	border-radius: 5px;
	background-position:center;
	background-color: rgb(255, 85, 0);
}
QPushButton:hover{
	background-color: rgba(50, 50, 50, 50);
	
	color: rgba(200, 200, 200, 200);
}

QPushButton:pressed{
	background-color: rgba(35, 35, 35, 35);
	
	color: rgba(200, 200, 200, 200);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_tela_login">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>1280</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 238);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEdit_user">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>320</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(45,45,45,45)
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30,30,30,30);
	rgba(100, 100, 100, 100)

}

</string>
    </property>
    <property name="placeholderText">
     <string>Matr√≠cula</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_senha">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>320</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
	border: 2px solid rgb(45,45,45,45)
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30,30,30,30);
	rgba(100, 100, 100, 100)

}

</string>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_entrar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>320</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	
	background-color: rgb(51, 51, 51);
	border: 2px solid rgb(60,60,60);
	border-radius: 5px;
	color: rgb(255, 255, 255);
}
QPushButton:hover {
	background-color: rgb(60,60,60);
	border: 2px solid rgb(70,70,70);
}



QPushButton:pressed {
	background-color: rgb(120,120,120);
	border: 2px solid rgb(90,90,90);
}
</string>
    </property>
    <property name="text">
     <string>Entrar</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/face/teste.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_txt_Login">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_txt_entrarnaconta">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Entre na sua conta</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="files_rc.qrc"/>
 </resources>
 <connections/>
</ui>
